packages/cli/src/tools/errors.js

```diff
@@ -2,10 +2,53 @@
  * @flow
  */
 import chalk from 'chalk';
+import dedent from 'dedent';
 
 export class ProcessError extends Error {
   constructor(msg: string, processError: string) {
     super(`${chalk.red(msg)}\n\n${chalk.gray(processError)}`);
     Error.captureStackTrace(this, ProcessError);
   }
 }
+
+type JoiErrorT = {
```

Comment:
maybe it would be easier to `flow-typed install joi` and use something from there?

Reply:
Yeah, that makes more sense.